src/Utils/MultiStepInput.ts

```diff
@@ -57,12 +57,12 @@ interface InputBoxParameters {
 
 /* istanbul ignore next */
 class MultiStepInput {
-  static async run<T>(start: InputStep) {
+  static async run(start: InputStep) {
```

Comment:
@jyoungyun PTAL :-D I guess this generic is not used at all.

Reply:
As I know, this code came from the vscode example. Do we have to fix?

Reply:
@jyoungyun I think the examples are not the best practices. I also found some unnecessary codes or a bit weird implementations.

Reply:
I'm not saying that the example is perfect, but I mean that it's easier to manage the code in future not to modify the upstream code. If a change occurs in the upstream code, we will have to consider more things rather than apply them as they are. :)

Reply:
But I doubt there will be an update related to that generic. The example repository does look like appending more example codes for other devs, rather than maintaining its existing project... aside from the fact that it's not been updated since 2020 May.

And if we're really gonna use the generics, then we can adapt that on the spot.